src/test/resources/ebean.properties

```diff
@@ -160,8 +160,8 @@ ebean.sqlserver.databasePlatformName=sqlserver17
 
 datasource.sqlserver.username=test_ebean
 datasource.sqlserver.password=SqlS3rv#r
-datasource.sqlserver.databaseUrl=jdbc:sqlserver://localhost:1433;databaseName=test_ebean
-datasource.sqlserver.databaseDriver=com.microsoft.sqlserver.jdbc.SQLServerDriver
+datasource.sqlserver.url=jdbc:sqlserver://localhost:1433;databaseName=test_ebean;sendTimeAsDateTime=false
```

Comment:
Ah, the sendTimeAsDateTime flag ;) https://github.com/microsoft/mssql-jdbc/pull/798